src/test/javascript/spec/component/programming-exercise/programming-exercise-import.component.spec.ts

```diff
@@ -38,8 +41,22 @@ describe('ProgrammingExerciseImportComponent', () => {
 
     beforeEach(async () => {
```

Comment:
As above, I think we can get rid of the `async` here

Reply:
Moreover, I cannot comment the [line ](https://github.com/ls1intum/Artemis/pull/3940/files#diff-2cef53c0606c7c16c2cda93fb34552b91e24798b3916102ceb41f6aa0e338e88R77-R79)
```
    afterEach(() => {
        pagingStub.restore();
    });
````
can be changed to:
````
    afterEach(() => {
        sinon.restore();
    });
````

to avoid that if more stubs are introduced, those are not restored.

Reply:
Agree :) Adapted in 10c9c74